src/components/Layout/Header/Header.js

```diff
@@ -0,0 +1,104 @@
+import { Button } from "@chakra-ui/button";
+import { SearchIcon } from "@chakra-ui/icons";
```

Comment:
zastanawiam się czy skoro dodałeś react icons to nie dać sobie spokój z ikonami z chakry (żeby nie wprowadzać niepotrzebnego dualizmu)- z tego co widzę to react icons jest dużo bogatsze

Reply:
teoretycznie icony z chary już są więc wydaje mi się, że to już znikoma optymalizacja, ale ok ;)